internal/bus/topics.go

```diff
@@ -6,18 +6,16 @@
 package bus
 
 const (
-	InstanceConfigUpdateRequestTopic = "instance-config-update-request"
-	InstanceConfigUpdateStatusTopic  = "instance-config-update-status"
-	InstanceConfigContextTopic       = "instance-config-context"
-	MetricsTopic                     = "metrics"
-	ConfigClientTopic                = "config-client"
-	AddInstancesTopic                = "add-instances"
-	UpdatedInstancesTopic            = "updated-instances"
-	DeletedInstancesTopic            = "deleted-instances"
-	ResourceUpdateTopic              = "resource-update"
-	NginxConfigUpdateTopic           = "nginx-config-update"
-	InstanceHealthTopic              = "instance-health"
-	ConfigUploadRequestTopic         = "config-upload-request"
-	DataPlaneResponseTopic           = "data-plane-response"
-	ConnectionCreatedTopic           = "connection-created"
+	AddInstancesTopic                 = "add-instances"
+	UpdatedInstancesTopic             = "updated-instances"
+	DeletedInstancesTopic             = "deleted-instances"
+	ResourceUpdateTopic               = "resource-update"
+	NginxConfigUpdateTopic            = "nginx-config-update"
+	InstanceHealthTopic               = "instance-health"
+	ConfigUploadRequestTopic          = "config-upload-request"
+	DataPlaneResponseTopic            = "data-plane-response"
+	ConfigApplyRequestTopic           = "config-apply-topic"
+	ConfigApplySuccessfulRequestTopic = "config-apply-successful-topic"
+	ConfigApplyFailedRequestTopic     = "config-apply-failed-topic"
```

Comment:
```suggestion
	ConfigApplyRequestTopic           = "config-apply-request"
	ConfigApplySuccessfulRequestTopic = "config-apply-successful-request"
	ConfigApplyFailedRequestTopic     = "config-apply-failed-request"
```

Reply:
I've changed these in Rollback for Failed and success no longer have request in the name, can I change ConfigApplyRequestTopic  there instead to avoid issues ?